features/networking/operator.feature

```diff
@@ -23,9 +23,12 @@ Feature: Operator related networking scenarios
     Given the master version >= "4.0"
     #Getting OCP version
     Given evaluation of `cluster_version('version').version` is stored in the :ocp_version clipboard
-    #Making sure that operator is not Failing before proceesing further steps
-    And evaluation of `cluster_operator('network').condition(type: 'Failing')` is stored in the :failing_status_before_patch clipboard
-    Then the expression should be true> cb.failing_status_before_patch["status"]=="False"
+    #Making sure that operator is not Degraded before proceesing further steps
+    And evaluation of `cluster_operator('network').condition(type: 'Degraded')` is stored in the :Degraded_status_before_patch clipboard
+    Then the expression should be true> cb.Degraded_status_before_patch["status"]=="False"
+    #Making sure that operator is not Degraded before proceesing further steps
+    And evaluation of `cluster_operator('network').condition(type: 'Degraded')` is stored in the :Degraded_status_before_patch clipboard
```

Comment:
as a matter of style could you use lower case in clipboard names?

Reply:
other than that looks good to me

Reply:
Thanks for catching that @akostadinov . Fixed it. Didn't mean to make them capital but seems like :%s// operation in VI editor made it along with others.